README.md

```diff
@@ -113,12 +113,12 @@ So it might be needed to guard certain features / workaround by the available se
 
 ## Releasing
 
-1. Run `bumpversion release`
-1. Generate the changelog (eg `towncrier --yes --version 0.1.0`)
-1. Commit your local changes, open a PR, and get it merged.
+1. Generate the changelog (eg `towncrier --yes --version 0.1.0`) and commit
```

Comment:
With this parameter towncrier is obviously not guessing the version, but could it?

Reply:
The problem is that the changelog commit needs to be independent of the commit that sets the release version, and the changelog commit should proceed the release commit. It's possible the workflow could be:

1. Run `bumpversion release`
2. Generate the changelog (eg `towncrier --yes --version 0.1.0`) and commit only the changelog
3. Commit the other files

I don't know that this is any simpler.